csharp/RowOriented/ParquetRowWriter.cs

```diff
@@ -61,6 +109,15 @@ private ParquetRowWriter(ParquetFileWriter parquetFileWriter, WriteAction writeA
             _rows = new TTuple[1024];
         }
 
+        private ParquetRowWriter(ParquetFileWriter parquetFileWriter, WriteAction writeAction, LogicalWriteConverterFactory? logicalWriteConverterFactory = null)
```

Comment:
I think you can remove the original private constructor above and just keep this one. Because it's private there's no concern about binary compatibility here

Reply:
I think I understand how binary compatibility works now, thanks!